src/docs/product/accounts/membership.mdx

```diff
@@ -15,51 +15,66 @@ which can then join one or more organizations. Each user should have
-| Can change Organization Settings                             |         |        |       | X       | X     |
-| Can remove an Organization                                   |         |        |       |         | X     |
-
-## Roles
+### Organization Member Roles
 
 Member roles dictate access within an organization.
 
-### Owner
+| Action                                                       | Billing | Member | Manager | Owner |
+| ------------------------------------------------------------ | ------- | ------ | ------- | ----- |
+| Can see/edit billing information and subscription details    | X       |        |         | X     |
+| Can see/edit legal and compliance details                    | X       |        |         | X     |
+| Can view and act on issues, such as assigning/resolving/etc. |         | X      | X       | X     |
+| Can join and leave teams                                     |         | X      | X       | X     |
+| Is automatically a Team Admin                                |         |        | X       | X     |
+| Can add/remove teams                                         |         |        | X       | X     |
+| Can change Organization Settings                             |         |        | X       | X     |
+| Can remove an Organization                                   |         |        |         | X     |
+
+#### Owner
 
 - Unrestricted access to the organization, its data, and settings.
 - Can add, modify, and delete projects and members, as well as manage subscription and billing details.
 - Can manage legal and compliance details.
 - Can delete an organization.
 
-### Manager
+#### Manager
 
 - Gains admin access on all teams.
 - Can add and remove members from the organization.
-
-### Admin
-
-- Admin access only on teams they're a member of.
 - Can create new teams and projects, as well as remove teams and projects they're a member of.
 
-### Member
+#### Member
 
 - Can view and act on events, as well as view most other data within the organization.
 
-### Billing
+#### Billing
 
 - Can manage subscription and billing details.
 - Can manage legal and compliance details.
 
+### Team Member Roles
+
+Team roles dictate access within projects owned by the team.
+
+A project may belong to multiple teams. To have administrative access to a
+project, you only need to be an admin of one of those teams, not all.
+
+| Action                       | Team Contributor | Team Admin |
+| ---------------------------- | ---------------- | ---------- |
+| Can change Project Settings  |                  | X          |
+| Can add/remove projects      |                  | X          |
+| Can edit Global Integrations |                  | X          |
+| Can add/remove Repositories  | X                | X          |
+
+#### Team Admin
+
+- Can create new projects and remove projects they're a member of.
+
+#### Team Contributor
+
+- Can view and act on events, as well as view most other data within the team's
+  projects.
```

Comment:
I wrote these for symmetry with the "Organization Member Roles" section, but they seem a little unnecessary and can maybe just be deleted?

Reply:
I would be fine with that.

Reply:
Done.